src/input.c

```diff
@@ -349,6 +401,16 @@ static void csi_key(input_t *restrict input, char ch, int parm1, int parm2)
         bool already_held = is_key_held(input, doom_key);
         bool is_new_keypress = false;
 
+        /* handle kitty release event */
+        if (input->kitty_keyboard_protocol_active) {
+            if (input->csi_u_params.event_type == 1) {
+                doom_key_down(doom_key);
+            } else if (input->csi_u_params.event_type == 3) {
+                doom_key_up(doom_key);
+            }
```

Comment:
Event_type 2 (repeat) is not handled.

Either handle explicitly or add comment explaining why ignored.